Amazon.QLDB.Driver/Amazon.QLDB.Driver.csproj

```diff
@@ -34,7 +34,7 @@
     <PackageReference Include="AWSSDK.QLDBSession" Version="3.5.1.4" />
```

Comment:
I checked the minimum versions of Ion and think they're reasonable. Can we figure out the minimum SDK version?

Reply:
Updated minimum SDK version to 3.5.1. I verified this is the earliest SDK version that support query stats.